src/features/unified-share-modal/InviteePermissionsMenu.js

```diff
@@ -114,6 +114,8 @@ class InviteePermissionsMenu extends Component<Props> {
             plainButton
         );
 
+        // TODO: `DropdownMenu` doesn't currently handle a scenario where the menu is taller than
```

Comment:
Right now, `DropdownMenu` has two behaviors: to open the content below the menu item, or above it if one of the constraints is violated. in the case where neither would cause the menu to fully display, both behaviors cause issues.

Perhaps the better place for this would be an issue here describing the change to make in `DropdownMenu`, but not sure what exactly would be the action.

# should it use `ScrollWrapper` and a fixed height?
# should it appear adjacent to the menu target?
# should it have native behavior and expand upward as a user traverses down into the available option?

I didn't want to prescribe the change to make, since I'm not sure which approach is best. Suggestions?